package.json

```diff
@@ -46,6 +46,9 @@
     "postversion": "git push --tags origin release-$npm_package_version",
     "postinstall": "patch-package && playwright install chromium"
   },
+  "resolutions": {
+    "**/elliptic": "6.6.1"
```

Comment:
why ** here? cant we import something specific?

Reply:
The ** forces the version for all dependencies to prevent any dependency from trying to import an earlier version